server/src/db_manager.py

```diff
@@ -267,7 +267,7 @@ def add_bill(self, payer : str, creator_username : str, title : str, total : flo
                 c.execute("""insert into payments
                     (bill_id, payee_name, amount_owed, is_paid) 
                     values 
-                    (?, ?, ?, ?)""", (next_id[0], payment.payee_name, payment.amount_owed, payment.is_paid))
+                    (?, ?, ?, ?)""", (next_id[0], payment['person'], payment['amount'], False))
```

Comment:
instead of changing the db_manager function to process the dict, can you convert the incoming data to payment objects in the create_bill endoint? Right now this breaks the unit test

something like:
```
payment_objects = []
for payment in outstanding payments:
   payment_obj = Payment(*(payment[person], payment[amount]))
   payment_objects.append(payment_obj)
self.database.add_bill(payment_objects)
```

Reply:
Agreed, please use given function and data object as is